pyproject.toml

```diff
@@ -127,6 +127,9 @@ src = ["earthaccess", "stubs", "tests"]
 [tool.ruff.lint]
 extend-select = ["I", "T20"]
 
+[tool.ruff.lint.per-file-ignores]
+"earthaccess/search.py" = ["T20"]
```

Comment:
I do not know if this is necessarily what we want but I didn't know how to ignore that we have intentional print statements 

> earthaccess/search.py:312:9: T201 `print` found
> earthaccess/search.py:313:9: T201 `print` found

Reply:
On those lines, you can append a comment like `# noqa: T201` :) https://docs.astral.sh/ruff/linter/#error-suppression